src/iccpd/src/iccp_cmd_show.c

```diff
@@ -449,6 +489,116 @@ int iccp_local_if_dump(char * *buf,  int *num, int mclag_id)
                     return EXEC_TYPE_FAILED;
             }
         }
+
+        if (csm->peer_link_if) {
+
+            lif_peer = csm->peer_link_if;
+
+            memset(&mclagd_lif, 0, sizeof(struct mclagd_local_if));
+
+            mclagd_lif.ifindex = lif_peer->ifindex;
+
+            if (lif_peer->type == IF_T_UNKNOW)
+                memcpy(mclagd_lif.type, "Unknown", 6);
+            else if (lif_peer->type == IF_T_PORT)
```

Comment:
wouldn't it be better to use strlen() instead of trying to count the characters manually?  very easy to introduce error...

Reply:
@gechiang : Thanks , Corrected.!